components/HomePage/UserHomePage.tsx

```diff
@@ -1,12 +1,34 @@
-import React from "react";
+import React, { useState } from "react";
 import { UserCalendar } from "../Calendar/UserCalendar";
+import { AvailabilityModal } from "../ManageAvailabilityWizard/AvailabilityModal";
+import styles from "./AdminHomePage.module.css"; //TODO change to user specific
 
 type UserHomeProp = {
   userId: string;
 };
 
 const UserHomePage: React.FC<UserHomeProp> = ({ userId }) => {
-  return <UserCalendar userId={userId} />;
+  const [showManageAvailability, setShowManageAvailability] = useState(false);
+
+  const handleClose = (): void => {
+    setShowManageAvailability(false);
+  };
+
+  return (
+    <div>
+      <div className={styles.homeWrapper}>
+        <button className={styles.createBtn} onClick={() => setShowManageAvailability(true)}>
+          Manage
+          <img className={styles.addIcon} src="AddIcon.png" />
+        </button>
+        {showManageAvailability ? (
+          <AvailabilityModal handleClose={handleClose} userId={userId} />
+        ) : null}
+      </div>
+
+      <UserCalendar userId={userId} />
+    </div>
+  );
```

Comment:
instead of this being a user home page, we need to make this a TeacherHomePage, that only gets rendered if the user's role is a teacher